lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -857,6 +924,24 @@ class HybridTextEditing {
       _emptyCallback,
     );
   }
+
+  void sendTextConnectionClosedToFlutterIfAny() {
```

Comment:
nit: I'd replace "Flutter" with "Framework". Engine is also Flutter, so it's not clear where exactly the message is being sent.

Reply:
This is probably my fault. I used the "send/sync to flutter" wording in the early version of text editing.

Reply:
I know. We're just passive-aggressively fixing your code ;)

Reply:
done